sql/create-db.sql

```diff
@@ -89,6 +89,7 @@ CREATE TABLE IF NOT EXISTS cloudwatchlog (
 -- Amazon Machine Image (ami) usage
 CREATE TABLE IF NOT EXISTS amiusage (
   region VARCHAR(128) NOT NULL, -- ec2 region
-  id VARCHAR(128) NOT NULL,
-  lastused TIMESTAMPTZ NOT NULL,
+  id VARCHAR(128) NOT NULL, -- opaque ID per Amazon
```

Comment:
were these comments added intentionally?

Reply:
Yes, I noticed all of the other tables also had comments so I didn't see any harm in it

Reply:
over commenting is always better than under commenting, provided the comments are correct :)